src/main/resources/static/login.html

```diff
@@ -22,8 +22,10 @@ <h1>HarmonIQ</h1>
 
         <div class="signin-container">
             <button class="signin-button">
+                <a href="/login">
```

Comment:
Doesn't this cause a circular path error because the static file's name is `login.html` and also there is a endpoint at `/login`?

Reply:
No it works, if I make it <a href="/login.html"> and make the endpoint "/login.html" then its a circular path error.